### PR TITLE
Check for zipped multi xlsx files and refactor

### PR DESCRIPTION
**Pull request by luciipak**

### CODE REVIEW

File: jobs/pulse3d/manifests/overlays/prod/kustomization.yaml [hunk 1]
```diff
@@ -7,192 +7,114 @@ resources:
...
-      path: /spec/jobTargetRef/template/spec/containers/0/image
-      value: "245339368379.dkr.ecr.us-east-2.amazonaws.com/pulse3d-worker:0.32.4"
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/1/value
-      value: prod-pulse3d-uploads
-    - op: replace
-      path: /spec/jobTargetRef/template/spec/containers/0/env/2/value
-      value: prod-mantarray-logs
\ No newline at end of file
+  - target:
```
Comment (tannermpeterson) [inline jobs/pulse3d/manifests/overlays/prod/kustomization.yaml:10]: Looks like a formatter ran on this, a little hard to tell the diff

Comment (luciipak) [reply jobs/pulse3d/manifests/overlays/prod/kustomization.yaml:10]: it was actually an annoying merge diff, vs code updated it's merge editor so I accidentally just kept manually changed the diff.
only change is removal of all old pulse3d versions

Comment (tannermpeterson) [reply jobs/pulse3d/manifests/overlays/prod/kustomization.yaml:10]: Can we remove the indents? I think we saw the indents cause issues before for some reason. It was either Kubernetes or argo that didn't like them

Comment (luciipak) [reply jobs/pulse3d/manifests/overlays/prod/kustomization.yaml]: yep, done

**Review by tannermpeterson — APPROVED** [review]

**Pull request merged**